src/main/java/seedu/address/model/module/Module.java

```diff
@@ -0,0 +1,88 @@
+package seedu.address.model.module;
+
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
+
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.Objects;
+import java.util.Set;
+
+import seedu.address.model.person.Person;
+
+/**
+ * Represents a module in FaculType.
+ * Guarantees: details are present and not null, field values are validated, immutable.
+ */
+public class Module {
+
+    // Identity fields
+    private final Code code;
+
+    // Data fields
+    private final Name name;
+    private final Set<Person> persons = new HashSet<>();
```

Comment:
Perhaps it is better to rename the set to "professors"?

Reply:
Hm... I was just following the class name. Wouldn't it be weird if "professors" suddenly appeared out of nowhere since it has never been mentioned before?

Reply:
You are right, I think just leave it as "person" for now.

Reply:
LGTM for me 👍